readthedocsext/theme/templates/projects/project_pull_requests.html

```diff
@@ -0,0 +1,31 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+{% load static %}
+{% load crispy_forms_filters %}
+
+{% block title %}{% trans "Documentation pull requests" %}{% endblock %}
+
+{% block project_pull_requests_active %}active{% endblock %}
+{% block project_edit_content_header %}{% trans "Documentation pull requests" %}{% endblock %}
```

Comment:
I'd match the navigation title on these:

```suggestion
{% block title %}{% trans "Pull request builds" %}{% endblock %}

{% block project_pull_requests_active %}active{% endblock %}
{% block project_edit_content_header %}{% trans "Pull request builds" %}{% endblock %}
```